packages/expo-camera/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### 🎉 New features
 
 - On iOS added new `codec` parameter in `recordAsync()` method and new method `getAvailableVideoCodecsAsync()` that queries the device for available video codecs. ([#12772](https://github.com/expo/expo/pull/12772) by [@ajsmth](https://github.com/ajsmth))
+- On iOS added new `requestCameraPermissions()` and `requestMicrophonePermissions()` methods which gives more fine-grained control over requested permissions. ([#12860](https://github.com/expo/expo/pull/12772) by [@ajsmth](https://github.com/ajsmth))
```

Comment:
```suggestion
- Added new `requestCameraPermissionsAsync()`, `requestMicrophonePermissionsAsync()`, `getCameraPermissionsAsync()` and `getMicrophonePermissionsAsync()` methods which gives more fine-grained control over requested permissions. ([#12860](https://github.com/expo/expo/pull/12772) by [@ajsmth](https://github.com/ajsmth))
```

Reply:
Good call - although `requestPermissionAsync()` for iOS has the same behaviour - only requests access to camera, the microphone prompt only comes up when audio capture begins (e.g `recordAsync()`) is called

Reply:
also `recordAsync()` is not available on web so this method is just to keep parity (no need for microphone permissions) - although this is commented out as a TODO so maybe I will look into tackling video capture for web at some point!